features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import ClusterStory from "../../components/clusterStory/clusterStory.jsx";
+//import { ExplorerContext } from "../../context/explorer-context.jsx";
+import i18n from "../../i18n.js";
+import SectionTitle from "../../components/clusterStories/sectionTitle.jsx";
+import Tab from "../../components/clusterStories/tab.jsx";
+
+import "./digitalGiantsStory.css";
+import BulletPoints from "../../components/bulletPoints/bulletPoints.jsx";
+
+const i18nHeader = "digitalGiantsStory";
```

Comment:
Good factoring out, which makes me think of maybe changing the interface of i18n to make this easier.

Reply:
Created [this issue](https://jira.polypoly.eu/browse/PROD4POD-1194) to take care of that.